Dockerfile

```diff
@@ -1,15 +1,22 @@
-FROM node:10
+# Use alpine node version
+FROM node:10.15.3-alpine
 
-# Create app directory
-WORKDIR /usr/local
+# Create working directory and set node user as owner
+WORKDIR /usr/src/app
+RUN chown node:node /usr/src/app
+
+# Switch to node user
+USER node
 
 # Install app dependencies
 # A wildcard is used to ensure both package.json AND package-lock.json are copied
 # where available (npm@5+)
-COPY package*.json ./
-
+COPY --chown=node:node package*.json ./
 RUN npm install
+# If you are building your code for production
+# RUN npm ci --only=production
 
-RUN selenium-standalone start
+# Bundle app source setting node user as owner
+COPY --chown=node:node . .
```

Comment:
The copy above copies the package.json file only.  This copies the rest of the application code.  

You could do it as one command and copy all at once, but this will copy in different layers which means that if you're building the image multiple times over it won't recreate layers that haven't changed.  So building the container wouldn't need to run npm install again if just the code changed.

It might also be worth considering changing this to use yarn instead as that's what the readme is suggesting should be used in stead of npm.